### DISCUSSION TITLE
Feature request to integrate rclone with gbm

### DISCUSSION THREAD

**Discussion by Aeropro4** [MikeMaximus/gbm#307] [Ideas]
Please integrate rclone with gbm like ludusavi which enables us to save save game to cloud storage. It would be more safe and protected from unforeseen events.

**Comment by MikeMaximus** [comment]
You can use whatever software you wish to sync the GBM backup folder with your cloud storage.  Dropbox, OneDrive, rclone, etc.  I'm not planning to ever directly integrate support for any specific cloud services or applications.

You can use the rclone sync to sync the GBM backup folder with your cloud storage whenever you need to or use a script that does it on a schedule.

**Comment by Aeropro4** [comment]
> You can use whatever software you wish to sync the GBM backup folder with your cloud storage.  Dropbox, OneDrive, rclone, etc.  I'm not planning to ever directly integrate support for any specific cloud services or applications.
> 
> You can use the rclone sync to sync the GBM backup folder with your cloud storage whenever you need to or use a script that does it on a schedule.

Hmm OK
